boards/pjrc/teensy4/doc/index.rst

```diff
@@ -221,9 +258,12 @@ Only Teensy 4.1:
 +-----+------------+-------------------------------------+
 | 40  | AD_B1_04   | GPIO1_20                            |
 +-----+------------+-------------------------------------+
-| 41  | AD_B1_05   | GPIO1_21                            |
+| 41  | AD_B1_05   | GPIO1_21 / UART3_RX                 |
 +-----+------------+-------------------------------------+
 
+Only Teensy Micromod
+
+
```

Comment:
It seems like you would need to restore this, unless you meant to change the pin definition for the Teensy 4.1 too (in which case a separate commit would be appreciated).

Reply:
Sorry, the last part of that change I removed that last only Teensy Micromod from the file

Reply:
No problem, thanks for the edit.